{{cookiecutter.project_slug}}/src/features/{% if cookiecutter.has_messenger_blueprint == "y" %}Messenger{% endif %}/screens/ChatScreen/index.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import { GiftedChat } from 'react-native-gifted-chat'
+import PubNubReact from 'pubnub-react';
+
+export default class ChatScreen extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.pubnub = new PubNubReact({
+      publishKey: 'pub-c-09038c51-029c-47a7-95f2-b6c7407afb12',
+      subscribeKey: 'sub-c-6b1bee04-ba9f-11e9-8753-ce76e7dc5905'
+    });
+    this.pubnub.init(this);
+    state = {
+     messages: [],
+    }
+    this.id = this.randomid();
+  }
+  randomid = () => {
+    return Math.floor(Math.random() * 100);
+  };
+
+  componentWillUnmount() {
+    this.pubnub.unsubscribe({ channels: ['ReactChat'] });
+  }
+
+  componentWillMount() {
```

Comment:
`componentWillMount` is deprecated, you might want to change this to `componentDidMount`